packages/formatters/src/formatters.ts

```diff
@@ -373,6 +373,76 @@ export function messageLink<C extends Snowflake, M extends Snowflake, G extends
 	return `${guildId === undefined ? channelLink(channelId) : channelLink(channelId, guildId)}/${messageId}`;
 }
 
+/**
+ * Formats the content into a heading level one.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param content - The content to wrap
+ */
+export function headingLevelOne<C extends string>(content: C): `# ${C}` {
+	return `# ${content}`;
+}
+
+/**
+ * Formats the content into a heading level two.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param content - The content to wrap
+ */
+export function headingLevelTwo<C extends string>(content: C): `## ${C}` {
+	return `## ${content}`;
+}
+
+/**
+ * Formats the content into a heading level three.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param content - The content to wrap
+ */
+export function headingLevelThree<C extends string>(content: C): `### ${C}` {
+	return `### ${content}`;
+}
+
+/**
+ * Formats the elements in the array to an ordered list.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param array - The array of elements to list
+ */
+export function orderedList<C extends string[]>(...array: C) {
+	return array.map((element, index) => `${index + 1}. ${element}`).join('\n');
+}
+
+/**
+ * Formats the elements in the array to an indented ordered list.
+ *
+ * @typeParam C - This is inferred by the supplied content
+ * @param array - The array of elements to list
+ */
+export function indentedOrderedList<C extends string[]>(...array: C) {
+	return array.map((element, index) => ` ${index + 1} ${element}`).join('\n');
```

Comment:
```suggestion
	return array.map((element, index) => ` ${index + 1}. ${element}`).join('\n');
```
by the looks of it you need a . here as well